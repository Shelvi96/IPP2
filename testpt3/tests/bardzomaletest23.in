NEW ID

64;9>786:3989:2
;1806;2>4100;4;98
759708;>88;:1249
:186526>12
;22:035>;45;54266
7>8896
;0>57249
298>1;86;125;1
:2;735>793
4582223>191
342:95>045;3083
6:7:43>:58
79584>32;
5442773>3::1:89
5;75;6>966
99::62>9:246524:
4>69;52;;88
377495>4;76130
@41:014:014;01
@24:980
@:49::;1
@176
@::368485706;221321105;:7:09
@;761698658358943
@56578:4704
@41441:9;05551922:65603
@;31726;3:1866645:12833
@96560
@930597737;107865
@;415089:9373745:8:;
@523483389900155166331;;874;86
@0;538
