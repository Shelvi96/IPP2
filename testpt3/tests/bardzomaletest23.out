0
0
0
0
152379577423872
28
0
130000020
120000011
0
28
30001
5733937602205373
0
